zone/lua_packet.cpp

```diff
@@ -80,7 +80,7 @@ void Lua_Packet::SetOpcode(int op) {
 
 int Lua_Packet::GetRawOpcode() {
 	Lua_Safe_Call_Int();
-	return static_cast<int>(self->GetOpcodeBypass());
+	return static_cast<int16>(self->GetOpcodeBypass());
```

Comment:
Why is this changed?

Reply:
This was my testing and forgot to return.  Sorry about that and great catch.